community-expectations.md

```diff
@@ -0,0 +1,85 @@
+## Community Expectations
+
+Kubernetes is a community project and, as such, it is wholly dependent on
+its community to provide a productive, friendly and collaborative environment.
+
+The first and foremost goal of the Kubernetes community to develop orchestration
+technology that radically simplifies the process of creating reliable
+distributed systems, however a second, equally important goal is the creation
+of a community that fosters easy, agile development of such an orcheastration
+systems.
+
+This document describes the community expectations for
+members of the Kubernetes community.  It is intended to be a living document
+that evolves as the community evolves.  It currently covers the expectations
+of conduct that govern all members of the community as well as the expectations
+around code review that govern all active contributors to Kubernetes.
+
+### Code of Conduct
+
+The most important expectation of the Kubernetes community is that all members
+abide by the Kubernetes [community code of conduct](code-of-conduct.md).
+Only by respecting each other can we develop a productive, collaborative
+community.
+
+### Code review
+
+As a community we believe in the value of code review for all contributions.
+Code review increases both the quality and readability of our codebase, which
+in turn produces high quality software.
+
+However, the code review process can also introduce latency for contributors
+and additional work for reviewers that can frustrate both parties.
+
+Consequently, as a community we expect that all active participants in the
+community will also be active reviewers.  To that end, we have set out the
+following expectations for code review:
+
+Any *active contributor* to the Kubernetes project is expected to be available
+to review code in areas where that contributor has expertise.  Active
+contributors or considered to be anyone who meets any of the following criteria:
+   * Sent more than two pull requests (PRs) in the previous one month, or more
```

Comment:
IF we're going to set exact numbers, we should double the margin in all directions.  Usually "a bar" is all that is required - if you said "1 PR in the last two months".

After the list, please describe the "why" of the list:

These criteria are intended to help us automate processes that deal with the large number of contributors to the project and helps us manage the project.  They are not intended to penalize members of the community with occasional contributions or who contribute in other spaces.

I.e. a bit more warm fuzzy.